teleport.js

```diff
@@ -324,3 +324,34 @@ function shot() {
     );
   }
 }
+
+/*
+ * teleports you to a random dnd tile in the same room.
+ */
+function dnd() {
+  const DND_TILE_NAME = "dnd-tile";
```

Comment:
i would put this outside with the const removed so it is a constant that is available on the console that the user can set themselves

Reply:
created issue for this. will do later